package.json

```diff
@@ -1,75 +1,68 @@
-    "@astrojs/react": "4.3.1",
-    "@emotion/cache": "11.14.0",
-    "@emotion/react": "11.14.0",
-    "@emotion/server": "11.11.0",
-    "@emotion/styled": "11.14.1",
-    "@mui/icons-material": "7.3.4",
-    "@mui/material": "7.3.4",
-    "astro": "5.13.9",
-    "react": "19.1.1",
-    "react-dom": "19.1.1"
-  },
   "engines": {
     "node": ">=20.3.0",
     "yarn": ">=1.22.21 <=1.22.22"
   },
+  "dependencies": {
+    "@emotion/react": "11.14.0",
+    "@emotion/styled": "11.14.0",
+    "@mui/icons-material": "6.3.0",
+    "@mui/material": "6.3.0",
+    "@vitejs/plugin-react-swc": "4.1.0",
+    "react": "18.3.1",
+    "react-dom": "18.3.1"
+  },
   "devDependencies": {
-    "@chromatic-com/storybook": "4.1.1",
-    "@eslint/js": "9.36.0",
-    "@storybook/addon-a11y": "9.1.10",
-    "@storybook/addon-docs": "9.1.10",
-    "@storybook/addon-vitest": "9.1.10",
-    "@storybook/blocks": "9.0.0-alpha.17",
-    "@storybook/react-vite": "9.1.10",
-    "@testing-library/jest-dom": "^6.9.1",
-    "@testing-library/react": "^16.3.0",
-    "@testing-library/user-event": "^14.6.1",
-    "@types/parse-json": "7.0.0",
-    "@types/prop-types": "15.7.15",
-    "@types/react": "19.1.13",
-    "@types/react-dom": "^19.2.0",
-    "@typescript-eslint/eslint-plugin": "8.44.0",
-    "@typescript-eslint/parser": "8.44.0",
-    "@vitejs/plugin-react": "^5.0.4",
-    "@vitest/browser": "3.2.4",
-    "@vitest/coverage-v8": "3.2.4",
-    "astro-eslint-parser": "1.2.2",
-    "eslint": "9.36.0",
+    "@chromatic-com/storybook": "3.2.2",
+    "@storybook/addon-a11y": "8.5.0",
+    "@storybook/addon-essentials": "8.5.0",
+    "@storybook/addon-interactions": "8.5.0",
+    "@storybook/addon-links": "8.5.0",
+    "@storybook/addon-onboarding": "8.5.0",
+    "@storybook/blocks": "8.5.0",
+    "@storybook/react": "8.5.0",
+    "@storybook/react-vite": "8.5.0",
+    "@storybook/test": "8.5.0",
+    "@storybook/test-runner": "0.23.0",
+    "@testing-library/jest-dom": "6.6.3",
+    "@testing-library/react": "14.3.1",
+    "@testing-library/user-event": "14.5.2",
+    "@types/react": "18.3.18",
+    "@types/react-dom": "18.3.5",
+    "@typescript-eslint/eslint-plugin": "8.20.0",
+    "@typescript-eslint/parser": "8.20.0",
+    "@vitejs/plugin-react": "4.3.4",
+    "@vitest/ui": "3.2.4",
+    "eslint": "9.18.0",
     "eslint-config-prettier": "10.1.8",
-    "eslint-import-resolver-node": "0.3.9",
-    "eslint-import-resolver-typescript": "4.4.4",
-    "eslint-plugin-astro": "1.3.1",
     "eslint-plugin-import": "2.32.0",
     "eslint-plugin-prettier": "5.5.4",
-    "eslint-plugin-storybook": "9.1.10",
-    "jsdom": "^27.0.0",
-    "playwright": "1.55.1",
+    "eslint-plugin-react-hooks": "5.1.0",
+    "eslint-plugin-react-refresh": "0.4.16",
+    "eslint-plugin-storybook": "0.11.1",
+    "jsdom": "25.0.1",
     "prettier": "3.6.2",
-    "storybook": "9.1.10",
-    "typescript": "5.9.2",
-    "vite": "7.1.9",
+    "storybook": "8.5.0",
+    "typescript": "5.7.2",
```

Comment:
Vite and related plugin versions look mismatched: `vite": "6.0.7"` together with `@vitejs/plugin-react` "4.3.4" and `@vitejs/plugin-react-swc`. These versions may be incompatible and break builds/tests.

**Sugerencia:**
```typescript
Align Vite and plugin versions according to their compatibility matrix. If you intend to use `@vitejs/plugin-react-swc`, prefer a Vite version compatible with that plugin and update TypeScript accordingly. Run a clean install and a local build to verify.
```